### PR TITLE
Add e2e test for dnat loadbalancer

### LINKED ISSUE #134: Add e2e tests for Nat LB type
**What would you like to be added**:
e2e tests need to be added for Nat type LB

**Why is this needed**:
to ensure the Nat type LB work well

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #134 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by chengxiangdong** [comment]
pls fix the ci-lint issues

**Comment by niuzhenguo** [comment]
> pls fix the ci-lint issues

done

File: test/e2e/natlbservice_test.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+			Namespace: namespace,
+			Name:      name,
+			Annotations: map[string]string{
+				"kubernetes.io/elb.class":     "dnat",
+				"kubernetes.io/natgateway.id": natID,
+			},
+			Labels: map[string]string{"app": "nginx"},
+		},
+		Spec: corev1.ServiceSpec{
+			ExternalTrafficPolicy: corev1.ServiceExternalTrafficPolicyTypeLocal,
```
Comment (chengxiangdong) [inline test/e2e/natlbservice_test.go:120]:
```suggestion
			LoadBalancerIP:        os.Getenv("HC_DNAT_IP"),
			ExternalTrafficPolicy: corev1.ServiceExternalTrafficPolicyTypeCluster,
```

File: test/e2e/natlbservice_test.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+					if err != nil {
+						return false, err
+					}
+					return false, nil
+				}, pollTimeout, pollInterval).Should(gomega.Equal(true))
+			})
+		}
+	})
+
+	ginkgo.It("service enhanced auto testing", func() {
```
Comment (chengxiangdong) [inline test/e2e/natlbservice_test.go:67]:
```suggestion
	ginkgo.It("dnat loadbalancer service testing", func() {
```

File: test/e2e/natlbservice_test.go (hunk 1 above)
Comment (niuzhenguo) [reply test/e2e/natlbservice_test.go]: done

File: test/e2e/natlbservice_test.go (hunk 2 above)
Comment (niuzhenguo) [reply test/e2e/natlbservice_test.go]: done

**Pull request merged**